README.md

```diff
@@ -69,6 +69,42 @@ const module = angular.module('app.changePassword', [])
 
 ```
 
+### Angular
+```js
+import { Component, OnInit } from '@angular/core';
+import { NGXLogger } from 'ngx-logger';
+import { Auth } from 'actions/auth';
+import { Notification } from 'utils/notification';
+
+@Component({
+    selector: 'app-login',
+    templateUrl: './login.component.html',
+    styleUrls: ['./login.component.scss'],
+})
+export class LoginComponent implements OnInit {
```

Comment:
We should change the name (and paths) to `ChangePassword` 😄

Reply:
Well that's the component, not module. Should I include module declaration as well, or just rename this component?

Reply:
Hmm you can rename the component and add a simple `App` module, like in the AngularJS example above.

Reply:
Sure, will do.